webapp/src/components/post_type_zoom/post_type_zoom.jsx

```diff
@@ -83,20 +83,36 @@ export default class PostTypeZoom extends React.PureComponent {
             if (this.props.fromBot) {
                 preText = `${this.props.creatorName} has started a meeting`;
             }
+            let announcementText = '';
+            if (props.meeting_host_joined) {
+                announcementText = 'Host already in the meeting';
+            } else {
+                if (props.meeting_waiting_count === 1) {
+                    announcementText = '1 participant waiting for host';
+                }
+                if (props.meeting_waiting_count > 1) {
```

Comment:
Can we do this for better readability?

```suggestion
                } else if (props.meeting_waiting_count > 1) {
```